sql/2020/06_Third_Parties/zero_third_party_websites.sql

```diff
@@ -0,0 +1,42 @@
+#standardSQL
+# Websites with no third party requests
+
+# Provides incorrect information in some cases, e.g. pageid = 140607555 
+
+WITH requests AS (
+  SELECT
+    pageid AS page,
+    req_host AS host
+  FROM
+    `httparchive.summary_requests.2020_08_01_mobile`
+),
+pages AS (
+  SELECT
+    pageid,
+    url
+  FROM
+    `httparchive.summary_pages.2020_08_01_mobile`
+),
+base AS (
+  SELECT
+    LOGICAL_AND(IF(NET.HOST(host) = NET.HOST(url), TRUE, FALSE)) zero_third_party,
```

Comment:
This is cool. Didn't know about `LOGICAL_AND`. But do you ned the IF ? TRUE : FALSE?
```suggestion
    LOGICAL_AND(NET.HOST(host) = NET.HOST(url)) zero_third_party,
```